src/base.mli

```diff
@@ -26,7 +26,7 @@ end]
 
 val track :
   event:string ->
-  ?properties:Properties.t ->
+  ?properties:(Properties.t[@js.default Properties.create []]) ->
```

Comment:
Il manque une espace.

Reply:
Un espace entre quels caractères? Car si c'est entre le "Properties.t" et le "[@js.default Properties.create []]" ce n'est pas moi qui ai décidé, le formatage automatique de "emacs" n'arrête pas de supprimer l'espace automatiquement à chaque fois que je sauvegarde et je ne peux rien n'y faire.

Reply:
Tu peux désactiver ou configurer le formatage automatique. Tu peux demander aux autres développeurs qui utilisent Emacs comment faire ça.

Reply:
Je pense que j'ai trouvé un compromis à court terme, j'ai édité en "hard"  via "gedit" et ça devrai passer, par contre dés que je remodifierai quoi que ce soi via gedit ça risque de refaire le coup...

Reply:
J'ai eu un retours du build, rajouter l'espace fait échouer la compilation. Je pense que cela viens de mon fichier ".ocamlformat" qui définit des règles à respecter concernant le code mais je ne vois pas comment autoriser ce cas de figure précis étant donné les options présentes:
<<
wrap-fun-args=false
let-binding-spacing=compact
field-space=loose
sequence-style=separator
doc-comments=before
margin=80
module-item-spacing=sparse
parens-tuple=always
parens-tuple-patterns=always
break-string-literals=auto
break-separators=after
space-around-lists=false
dock-collection-brackets=false
>>

Je pense qu'il faut modifier "let-binding-spacing"...

Reply:
Finalement, après expérimentation, ça ne peut pas êtres les valeurs que j'ai mis dans le ".ocamlformat" qui m'impossent de supprimer l'espace.
Le message d'erreur m'indique
<<
ocamlformat: "src/base.mli" was not already formatted. ([max-iters = 1])
>>

J'ai donc tenté de rajouter un paramètres "max-iters" avec une valeur supérieur à 1 mais ça n'a rien changé au fait que l'espace est coupé automatiquement, de plus ce n'est même pas logique car la documentation (https://github.com/ocaml-ppx/ocamlformat/blob/master/ocamlformat-help.txt) m'indique que la valeur de base de l'arguement est égale à 10...